code/modules/clothing/clothing.dm

```diff
@@ -106,6 +109,8 @@
 	..()
 	if(!istype(user))
 		return
+	for(var/trait in clothing_traits)
+		REMOVE_TRAIT(user, trait, "[CLOTHING_TRAIT] [REF(src)]")
```

Comment:
this doesn't check if it was in the slot that it should be in when removed

Also this would be nice as an atomized PR for clothing traits

Reply:
yeah I should probably remake this and make the traits a separate PR

Reply:
Removed all references to clothing_traits